src/lib/y2storage/proposal/space_maker.rb

```diff
@@ -298,21 +295,33 @@ def force_ptables(planned_partitions)
         end
       end
 
+      # Identifiers of all the devices that should be kept as-is, either because their are
+      # part of #protected_sids or because they would affect them indirectly
+      #
+      # @param devicegraph [Devicegraph] graph used to calculate the relationship between sids
+      # @return [Array<Integer>]
+      def all_protected_sids(devicegraph)
```

Comment:
Question: Could not a PV of a re-used VG be resized to make space for the booting partition?

Reply:
I added your comment to https://trello.com/c/88lh9u52/341-agama-redefine-the-biggerresize-approach-for-non-partitioned-disks where similar topics are discussed.